ruby_programming/basic_ruby/debugging.md

```diff
@@ -149,6 +151,40 @@ yell_greeting("bob")
 
 During the session, if you check for the value of `name`, you will notice that you get back the value `bob` instead of `BOB`. What value do you think `greeting` will return? Yup, it will be `nil`. This is because `name = name.upcase` and `greeting = "WASSAP, #{name}!"` occurred after the `binding.pry` call and were never evaluated.
 
+Using the same example above, you can use one of pry-byebug's commands to figure out what `name = name.upcase` will return. You won't need to quit the session or add another `binding.pry` beneath it. Enter `next` to step over to the next line.   
+
+~~~ruby
+     5: def yell_greeting(string)
+     6:   name = string
+     7: 
+     8:   binding.pry
+     9: 
+ => 10:   name = name.upcase
+    11:   greeting = "WASSAP, #{name}!"
+    12:   puts greeting
+    13: end
+
```

Comment:
In this code example, I would not duplicate the method. Instead start at line 11 of the example `[1] pry(main)> name`. Add a line for the value of `greeting` (from the previous instructions). Then continue with your instructions on `next`.